README.md

```diff
@@ -77,11 +77,42 @@ Use locally
 
 1. Add a `<script>` tag to your page for `collect.js`:
 
-        <script src="http://127.0.0.1:8000/static/collect.js"
-        data-webalyzer="example.com"></script>
+        <script>
+        (function() {
+        var s = document.createElement('script');
+        s.src = 'http://127.0.0.1:8000/static/collect.js';
+        document.head.appendChild(s); s.async=true; s.onload=function() {
+            Webalyzer('example.com');
+        }})();
+        </script>
 
 2. Hit your page(s)
 
 3. Go to the collected analysis page: http://127.0.0.1:8000/analyzer/
 
-4. Enter your `data-webalyzer` attribute value (e.g., `example.com`) and click "Start Analysis"
+There are configuration options available for when you initialize the
+`Webalyzer` in that script snippet. For example, instead of:
+
+        Webalyzer('example.com');
+
+You can do this:
+
+        Webalyzer('example.com', {
+             doSend: function(html, url) {
+                 if (getWeather() === 'raining') {
+                     return false;
+                 }
+                 return true;
+             },
+             beforeSendHTML: function(html, url) {
```

Comment:
We probably want a beforeSendCSS as well. Not a blocker for this PR though, additional feature.